docs/application/tizen-studio/release-notes/3-7-release-notes.md

```diff
@@ -0,0 +1,55 @@
+# Tizen Studio 3.7 Release Notes
+
+- Release Date: Mar 25, 2020
+
+## IDE and Tools
+
+### New Features
+
+- Bundeled JDK with Tizen Studio removing the dependency on system Java
+- Enhaced the security of certificates by adding support for SHA512 encryption
+- Support for Chrome 80 added
+- Log Viewer has been re enabled in the IDE
+
+### Fixed Bugs
+
+- Native unit test applications not getting installed on emulator
+
+### Known Issues
+
+- Tizen Studio
+  - In MacOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access. For example, for C:\ProgramFiles, administrator privileges are required to run the Tizen SDK tools. The Tizen Installer and the baseline SDK Installer alerts you, if you try to install into such a directory.
+- Web and Native IDE
+  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Now Tizen Studio does not support unit testing for older versions.
+- Web IDE
+  - The preview tab in the Web Page Editor sometimes does not appear properly. Use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.
+  - In Rapid Development Support (RDS) mode, the web unit test result is not updated.
+- Certificate Manager
+  - Overwriting a duplicate certificate profile in the migration wizard works incorrectly on macOS.
+- Emulator
+  - In 5.5 Emulator Images app un-installation might take some time to complete if the app name contains more than 14 characters. A patch for the same will be released in next update.
```

Comment:
Please see https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md
'images' is a noun.